mod to include larger headers (> 4KB)

larryboymi opened the pull request:
For issue #26 : You might want to make this be a config variable, I just hardcoded it.

tpbowden commented:
Thanks for the PR, but as you say I'd rather this were a CLI argument in the same way max request body size works before merging. Are you OK to add this extra change? If not I can do it when I have time at the weekend and you can continue running using a forked image.

larryboymi commented:
Yeah I can do it...but it'll be next week as I'm out of town. Thanks!

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
From: Tom Bowden <notifications@github.com>
Sent: Wednesday, December 14, 2016 3:03:02 AM
To: tpbowden/swarm-ingress-router
Cc: Larry Anderson; Author
Subject: Re: [tpbowden/swarm-ingress-router] mod to include headers up to 8KB (#28)


Thanks for the PR, but as you say I'd rather this were a CLI argument in the same way max request body size works before merging. Are you OK to add this extra change? If not I can do it when I have time at the weekend and you can continue running using a forked image.

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/tpbowden/swarm-ingress-router/pull/28#issuecomment-266980077>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ABT8KhLcNLeIp0yVJq44iL_4i6cKwU_4ks5rH7DGgaJpZM4LMMiF>.

larryboymi commented:
@tpbowden please check out da16d6b , where I updated the cli and tests to use a config var `read-buffer-size`. I added some comments from [fasthttp's Server](https://github.com/valyala/fasthttp) about the variable to `cli.go` (those can be removed if you want later).
